app/src/main/java/dev/arkbuilders/arkmemo/ui/adapters/NotesListAdapter.kt

```diff
@@ -206,16 +230,45 @@ class NotesListAdapter(
         notifyDataSetChanged()
     }
 
-    fun getNotes(): List<Note> {
+    fun getNotes(): MutableList<Note> {
```

Comment:
Why do we need mutable list here? We expect this list to change only within this class to have single source of truth

Reply:
Ok removed the mutability.